posthog/hogql/test/_test_parser.py

```diff
@@ -1348,13 +1348,13 @@ def test_select_limit_offset(self):
                 ),
             )
             self.assertEqual(
-                self._select("select 1 from events LIMIT 1 OFFSET 3 BY 1, event"),
+                self._select("select 1 from events LIMIT 1 BY event LIMIT 1 OFFSET 3"),
                 ast.SelectQuery(
                     select=[ast.Constant(value=1)],
                     select_from=ast.JoinExpr(table=ast.Field(chain=["events"])),
                     limit=ast.Constant(value=1),
                     offset=ast.Constant(value=3),
-                    limit_by=[ast.Constant(value=1), ast.Field(chain=["event"])],
+                    limit_by=ast.LimitByExpr(offset_value=ast.Constant(value=1), exprs=[ast.Field(chain=["event"])]),
```

Comment:
This test also doesn't seem right, we should have `n` here instead of `offset_value`